astropy/coordinates/builtin_frames/heliocentric.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, unicode_literals, division,
+                        print_function)
+
+from ..representation import SphericalRepresentation
+from ..baseframe import (BaseCoordinateFrame, RepresentationMapping,
+                         TimeFrameAttribute)
+from .utils import DEFAULT_OBSTIME
+
+class Heliocentric(BaseCoordinateFrame):
```

Comment:
@StuartLittlefair - this frame still appears to use RA/Dec, so we need a way to mention that in the name somehow. For instance, one could have a Galactic coordinates frame centered on the Sun. If this frame does use RA/Dec, which RA/Dec definition do you follow?